setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+CURRENT_USER=$USER
+
+# update package lists from repositories
+sudo apt-get update
+sudo apt-get install git
+
+sudo mkdir /usr/lib/jvm
```

Comment:
the same as above. Check first if java is present and if it is Oracle java. If present, then no need to install it again.